<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapOptionsDialog</class>
 <widget class="QDialog" name="MapOptionsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>249</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Set Map Options</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>315</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="MainLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="DescriptionLabel">
      <property name="text">
       <string>Description: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="DescriptionLE"/>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="MapDimensionsLayout">
      <item>
       <widget class="QSpinBox" name="WidthBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="DimensionsByLabel">
        <property name="text">
         <string>By</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="HeightBox">
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="DimensionsLabel">
      <property name="text">
       <string>Dimensions: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="MonsterGroupTitleLabel">
      <property name="text">
       <string>Mapwide Monster Group:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="MonsterGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="MonsterGroupLabel">
      <property name="text">
       <string>Monster Group: </string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="MonsterSettingsLayout">
      <item>
       <widget class="QLabel" name="MonsterChanceLabel">
        <property name="text">
         <string>Chance: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="MonsterChanceBox"/>
      </item>
      <item>
       <widget class="QLabel" name="MonsterDensityLabel">
        <property name="text">
         <string>Density: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="MonsterDensityBox"/>
      </item>
     </layout>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="ItemGroupTitleLabel">
      <property name="text">
       <string>Mapwide Item Group:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="ItemGroupBox"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="ItemGroupLabel">
      <property name="text">
       <string>Item Group:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <layout class="QHBoxLayout" name="ItemSettingsLayout">
      <item>
       <widget class="QLabel" name="ItemChanceLabel">
        <property name="text">
         <string>Chance: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="ItemChanceBox"/>
      </item>
      <item>
       <widget class="QCheckBox" name="ItemsOnGrassBox">
        <property name="text">
         <string>Spawn Items on Grass?</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MapOptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MapOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
